darts/models/forecasting/forecasting_model.py

```diff
@@ -802,6 +824,51 @@ def retrain_func(
                 logger,
             )
 
+        if fit_kwargs is None:
+            fit_kwargs = dict()
+        if predict_kwargs is None:
+            predict_kwargs = dict()
+
+        # sanity checks of the arguments directly exposed by historical_forecasts
+        if "predict_likelihood_parameters" not in predict_kwargs:
```

Comment:
I think here we can also just let python handle duplicated parameters.
It will throw an error:

```
SyntaxError: keyword argument repeated: predict_likelihood_parameters
```